src/functions-and-arrays.js

```diff
@@ -1,43 +1,109 @@
-// Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+// Iteration #1: Find the maximum OK
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a
+  } else if (a < b) {
+    return b
+  } else if (a === b) { 
```

Comment:
You don't need the `else if` statement, because if `a` and `b` are the same, you can just return `b`.